pkg/download/automation/automation.go

```diff
@@ -0,0 +1,109 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package automation
+
+import (
+	"encoding/json"
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/log"
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/maps"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/client/automation"
+	config "github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2/coordinate"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2/parameter"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2/parameter/value"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2/template"
+	v2 "github.com/dynatrace/dynatrace-configuration-as-code/pkg/project/v2"
+)
+
+var automationTypesToResources = map[config.AutomationType]automation.ResourceType{
```

Comment:
pkg/config and pkg/download/automation do not share the same type for resources, so we need a mapping

Reply:
this mapping is from pkg/config -> pkg/client/automation though right? As they're used right in the client calls

nitpick/maybe-bad suggestion: maybe importing `pkg/client/automation` as `automationClient` would make the distinction to this automation pkg clearer